crypto/rand/rand_lib.c

```diff
@@ -138,20 +148,23 @@ int RAND_poll(void)
      err:
         rand_pool_free(pool);
     }
-#endif
     return ret;
+# else
+    return 1;
```

Comment:
Is this the correct behaviour for this function, i.e. a no-op? Shouldn't it attempt to seed the provider based DRBG?

Reply:
Yep, this should just force reseeding.

Reply:
However this might be done in a different PR, if you want to. In that case please raise an issue against 3.0 milestone.

Reply:
I'm not convinced it's a great idea.  RAND_poll() in an idle loop could cause problems....

Still, 1.1.1 did reseed in this case, so I'll add it.